lib/browser/init.js

```diff
@@ -184,5 +184,20 @@ if (currentPlatformSupportsAppIndicator()) {
   process.env.XDG_CURRENT_DESKTOP = 'Unity'
 }
 
+// Quit when all windows are closed and no other one is listening to this.
+app.on('window-all-closed', () => {
+  if (app.listenerCount('window-all-closed') === 1) {
+    app.quit()
+  }
+})
+
+// Create default menu.
+app.once('ready', () => {
+  if (!Menu.getApplicationMenu()) {
```

Comment:
Your point in the call this morning gave me some thoughts on this, maybe there should be an explicit way to opt-out of this default without setting an empty menu.

E.g. Setting `process.noDefaultMenu` to `true` (similar to our `process.noAsar`)

Reply:
@nornagon, @codebytere what do you think?

Reply:
done